.gitignore

```diff
@@ -7,3 +7,4 @@ build/
 **/private.py
 .coverage
 dist/
+/pii_report/*.yaml
```

Comment:
Oddly enough, this isn't in any .gitignore files in edx and openedx repos. Maybe pii_check isn't hooked into test-all in other repos, or people don't use it, or they use `make clean` before going to commit.